docs/admin/runai-setup/self-hosted/ocp/backend.md

```diff
@@ -46,24 +45,51 @@ Perform the instructions for [local certificate authority](../../config/org-cert
 | `grafana.adminPassword`  | Grafana password  |   Override the Run:ai default password for accessing Grafana |
+| `grafana.dbUser`  | Grafana's username for PostgreSQL  |   Override the Run:ai default user name for Grafana to access Run:ai database (PostgreSQL) |
+| `grafana.dbPassword`  | Grafana's password for PostgreSQL |   Override the Run:ai default password for Grafana to access Run:ai database (PostgreSQL) |
+| `grafana.grafana.ini.database.user`  | Reference to Grafana's username for PostgreSQL  |  Don't override this value |
+| `grafana.grafana.ini.database.password`  | Reference to Grafana's password for PostgreSQL |   Don't override this value |
+| `tenantsManager.config.adminUsername`  | Run:ai first admin username |   Override the default user name of the first admin user created with Run:ai |
+| `tenantsManager.config.adminPassword`  | Run:ai first admin user's password |   Override the default password of the first admin user created with Run:ai |
 | `thanos.receive.persistence.storageClass` and `postgresql.primary.persistence.storageClass` | Storage class | The installation to work with a specific storage class rather than the default one |
-| `global.imagePullSecrets:` <br> &ensp; `- name: <secret-name>`  | Docker secret | Provide credentials for accessing the organization's docker registry. This is required for air-gapped environments  |
-| `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |   
-|<div style="width:200px"></div>| | |Use the `--set` syntax in the helm command above.  
+| `<component>` <br> &ensp;`resources:` <br> &emsp; `limits:` <br> &emsp; &ensp; `cpu: 500m` <br> &emsp; &ensp; `memory: 512Mi` <br> &emsp; `requests:` <br> &emsp; &ensp; `cpu: 250m` <br> &emsp; &ensp; `memory: 256Mi`  | Pod request and limits  |  `<component>` may be anyone of the following: `backend`, `frontend`, `assetsService`, `identityManager`, `tenantsManager`, `keycloakx`, `grafana`, `authorization`, `orgUnitService`,`policyService`  |
+|<div style="width:200px"></div>| | |
+
+Use the `--set` syntax in the helm command above.
+
+!!! Note
+    If you modify one of the usernames or passwords (KeyCloak, PostgreSQL, Grafana) after Run:ai is already installed, perform the following steps to apply the change:
+
+    1. Modify the username/password within the relevant component as well (KeyCloak, PostgreSQL, Grafana).
+    2. Run `helm upgrade` for Run:ai with the right values, and restart the relevant Run:ai pods so they can fetch the new username/password.
+
+## Next steps
```

Comment:
@jasonnovichRunAI this section was not changed by my PR.... if you changed it - ignore this comment, if not, this needs to get back to its original state